scripts/issue/488/transfer_balancer_tcl.py

```diff
@@ -0,0 +1,28 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    trops = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    vault = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+    trops.init_balancer()
+
+    bpt_20BTC_80_BADGER = trops.contract(registry.eth.balancer.B_20_BTC_80_BADGER)
+    bpt_20BTC_80_BADGER_GAUGE = trops.contract(
+        registry.eth.balancer.B_20_BTC_80_BADGER_GAUGE
+    )
+
+    trops.take_snapshot(
+        tokens=[bpt_20BTC_80_BADGER.address, bpt_20BTC_80_BADGER_GAUGE.address]
+    )
+    vault.take_snapshot(tokens=[bpt_20BTC_80_BADGER_GAUGE.address])
+
+    trops.balancer.stake_all(bpt_20BTC_80_BADGER)
+
+    bal = bpt_20BTC_80_BADGER_GAUGE.balanceOf(trops)
+
+    bpt_20BTC_80_BADGER_GAUGE.transfer(vault, bal)
+
+    vault.print_snapshot()
```

Comment:
```suggestion
    trops.print_snapshot()
    vault.print_snapshot()
```

Reply:
this is not strictly needed as the snapshot was taken lines above and will contain the `snapshot` attribute and this line will get exec: https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/great_ape_safe.py#L248

Reply:
ah sure. just a bit annoying to not have them next to each other to compare.